app/forms/oregon_digital/fields/input_factory.rb

```diff
@@ -1,26 +1,18 @@
 module OregonDigital
   module Fields
     class InputFactory
-      class << self
-        def create(object, property)
-          decorate do
-            base_factory.create(object, property)
-          end
-        end
-
-        private
+      pattr_initialize :base_factory, :decorator
 
-        def decorate
-          decorator.new(yield)
+      def create(object, property)
+        decorate do
+          base_factory.create(object, property)
         end
+      end
 
-        def decorator
-          HasHintOption
-        end
+      private
 
-        def base_factory
-          HydraEditor::Fields::Factory
```

Comment:
I don't know about this - we're turning an absolute must-have into an optional, configured parameter.  The core factory is absolutely essential to get the rest of the hydra editor's field functionality.